src/server/game/Handlers/ChannelHandler.cpp

```diff
@@ -56,6 +57,10 @@ void WorldSession::HandleLeaveChannel(WorldPacket& recvPacket)
     std::string channelName;
     recvPacket >> unk >> channelName;
 
+    //check for fake packets and bad addons that cause client to crash
+    if (!ChatHandler(this).isValidChatMessage(channelName.c_str()))
+        return;
```

Comment:
Why not make a small function for this? to avoid duplicate code

Reply:
lazzy and its old good code )

Reply:
oh ok :)

Reply:
@Viste can you please update this PR with this suggestion?

Reply:
yes after on next week